tracecat/registry/actions/models.py

```diff
@@ -390,7 +391,7 @@ def from_validation_result(
         )
 
     @staticmethod
-    def from_dsl_validation_error(exc: TracecatValidationError):
+    def from_dsl_validation_error(exc: RegistryValidationError):
```

Comment:
validation likely needs some consolidation, but i don't think this needs to be changed - this is for build-time validation, we explicitly catch `TracecatValidationError` exception groups. `RegistryValidationError` doesn't inherit from this so the returned struct will be malformed